packages/core/helper-plugin/lib/src/components/FilterListURLQuery/AttributeTag.js

```diff
@@ -42,6 +42,18 @@ const AttributeTag = ({ attribute, filter, onClick, operator, value }) => {
     formattedValue = formatNumber(value);
   }
 
+  // Handle custom input
+  if (attribute.metadatas.customInput) {
+    // If options, get the option label
+    if (attribute.metadatas.options) {
```

Comment:
Looks like you can combine the 2 if statements into 1

Reply:
I considered that, but I guess my idea was to be setup for any future conditions that fall under customInput. For now it could just be a single condition you're right

Reply:
I considered that, but I guess my idea was to be setup for any future conditions that fall under customInput. For now it could just be a single condition you're right